README.md

```diff
@@ -160,6 +160,9 @@ function getAllAddons({ categoryId }: GetAllAddonsParams = {}) {
   the keyboard. If it's something you think is unfixable then use
   `$FLOW_IGNORE` instead. Please explain your rationale in the comment and link
   to a GitHub issue if possible.
+* If you're stumped on why some Flow annotations aren't working, try using
```

Comment:
`flow` doesn't have to be global. It's available in `node_modules/.bin/` just like any other locally installed command. I have this in my `~/.bash_profile` and I highly recommend it:
```shell
# Get the bin for any local NodeJS module on cwd.
PATH="./node_modules/.bin:${PATH}"
```